.github/tools/cioutput.py

```
@@ -0,0 +1,669 @@
+		Returns:
+			Formatted string with ANSI color codes
+		"""
+		# Check if output is a terminal before using colors
+		use_colors = hasattr(sys.stdout, 'isatty') and sys.stdout.isatty()
+		is_boundry = getattr(record, "is_boundry", False)
+		# Format the basic message
+		message = super().format(record)
+		if is_boundry:
+			if len(message) > 0:
+				return f"[START] {message}"
+			else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)